.github/CODEOWNERS

```diff
@@ -110,7 +110,9 @@
 # Contributor Dashboard project.
 /core/controllers/contributor_dashboard*.py @sagangwee
 /core/domain/opportunity*.py @sagangwee
+/core/domain/translation*.py @vojtechjelinek
```

Comment:
Why is this assigned to @vojtechjelinek are we expecting it to be assigned to @sagangwee (Contributor dashboard team)?

Reply:
This was suggested by @seanlip.

Reply:
Yup, @vojtechjelinek is correct, I already checked with him. He is working on a project to migrate translations to their own models, so this fits in with that too.